ledger/src/blockstore.rs

```diff
@@ -6211,18 +6198,6 @@ pub mod tests {
             &last_root
         ));
 
-        // Trying to insert a shred with index < position should fail
-        {
-            let mut coding_shred = coding_shred.clone();
-            let index = coding_shred.index() - coding_shred.fec_set_index() - 1;
-            coding_shred.set_index(index as u32);
-
-            assert!(!Blockstore::should_insert_coding_shred(
-                &coding_shred,
-                &last_root
-            ));
-        }
```

Comment:
already tested in sanitize tests:
https://github.com/solana-labs/solana/blob/315707504/ledger/src/shred/legacy.rs#L417-L426